runtime/protocol/http-client-engines/http-client-engine-okhttp/jvm/src/aws/smithy/kotlin/runtime/http/engine/okhttp/OkHttpEngine.kt

```diff
@@ -69,6 +71,8 @@ private fun OkHttpEngineConfig.buildClient(): OkHttpClient {
         followRedirects(false)
         followSslRedirects(false)
 
+        connectionSpecs(listOf(minTlsConnectionSpec(config.minTlsVersion), ConnectionSpec.CLEARTEXT))
```

Comment:
**Question**: Why do we need to add `CLEARTEXT` here?

Reply:
We're replacing the default list of all acceptable connections, not just TLS ones. If we don't add `CLEARTEXT`, then HTTP connections will be impossible.